README.pt.md

```diff
@@ -1,94 +1,99 @@
-irc.freenode.net).
+[English](https://github.com/swaywm/sway/blob/master/README.md#sway--) -
+[日本語](https://github.com/swaywm/sway/blob/master/README.ja.md#sway--) - [Deutsch](https://github.com/swaywm/sway/blob/master/README.de.md#sway--) - [Ελληνικά](https://github.com/swaywm/sway/blob/master/README.el.md#sway--) - [Français](https://github.com/swaywm/sway/blob/master/README.fr.md#sway--) - [Українська](https://github.com/swaywm/sway/blob/master/README.uk.md#sway--) - [Italiano](https://github.com/swaywm/sway/blob/master/README.it.md#sway--) - [**Português**](https://github.com/swaywm/sway/blob/master/README.pt.md#sway--) -
+[Русский](https://github.com/swaywm/sway/blob/master/README.ru.md#sway--) - [Български](https://github.com/swaywm/sway/blob/master/README.bg.md#sway--)
 
-[![](https://sr.ht/ICd5.png)](https://sr.ht/ICd5.png)
+"**S**irCmpwn's **Way**land compositor" é um compositor [Wayland][], **em desenvolvimento**,
+compatível com o gerenciador de janelas [i3][].  Leia o [FAQ][sway faq].
+Participe do [canal no IRC][sway irc] (#sway em irc.freenode.net).
 
-Se você deseja apoiar o desenvolvimento do Sway, você pode contribuir com o
-SirCmpwn em sua [página no Patreon](https://patreon.com/sircmpwn) ou você
-pode colaborar com [premiações](https://github.com/swaywm/sway/issues/986)
-para recursos específicos. Qualquer um pode requerer uma premiação ao implementar
-o recurso especificado, e você pode criar uma premiação para qualquer recurso que desejar.
-O Patreon é melhor direcionado para a manutenção a longo prazo do Sway.
+Se desejas apoiar o desenvolvimento do projeto, podes contribuir com
+SirCmpwn em sua [página no Patreon][sircmpwn patreon].
 
 ## Ajuda em português
 
-No momento, o suporte em português no canal do IRC **não está ativo**. Em caso de problemas,
-use as [*issues*](https://github.com/swaywm/sway/issues/) do Github (*em inglês*).
+No momento, o suporte em português no canal no IRC **não está ativo**.
+Caso haja algum problema, abra uma [*issue*][sway issues]
+(*em inglês*).
 
-A tradução para português é um *trabalho em progresso*, no momento. Caso encontre algum erro
-ou queira colaborar com a tradução, visite
-[essa *issue*](https://github.com/swaywm/sway/issues/1318) para mais informações e não
-exite em enviar quaisquer correções necessárias.
+A tradução em português é um **trabalho em progresso**. Caso encontre
+erros ou queira colaborar com a tradução, visite [essa *issue*][sway i18n]
+para mais informações e não hesite em enviar quaisquer correções
+necessárias.
 
-## Assinaturas dos *Releases*
+## Assinaturas dos lançamentos
 
-*Releases* são assinadas com a chave
-[B22DA89A](http://pgp.mit.edu/pks/lookup?op=vindex&search=0x52CB6609B22DA89A)
-e publicadas [no GitHub](https://github.com/swaywm/sway/releases).
+Cada versão é assinada com a chave [B22DA89A][sway pgp] e publicada
+[no Github][sway releases].
 
 ## Instalação
 
 ### A partir de pacotes
```

Comment:
I  think `Por` is better than `A partir de` for this and subsequent subsections.